cds_ils/config.py

```diff
@@ -432,7 +432,36 @@ def _parse_env_bool(var_name, default=None):
 }
 
 ILS_CIRCULATION_MAIL_TEMPLATES = {
-    "librarian_footer": "cds_librarian_footer.html"
+    "librarian_footer": "cds_librarian_footer.html",
```

Comment:
I think this should disappear no? We can merge it and you remove it later on if you prefer...

Reply:
I kept it for now because it requires some refactor in `invenio-app-ils` if we want to remove it from there and add the footer to the `active_loans.html` template. IMHO it's better to remove it later